hs_core/views/discovery_json_view.py

```diff
@@ -0,0 +1,41 @@
+import simplejson as json
+from django.http import HttpResponse
+from haystack.query import SearchQuerySet
+from django import forms
+from haystack.forms import FacetedSearchForm
+from haystack.generic_views import FacetedSearchView
+from django.core import serializers
+from hs_core.discovery_form import DiscoveryForm
+
+class DiscoveryJsonView(FacetedSearchView):
+    facet_fields = ['author', 'subjects', 'resource_type', 'public', 'owners_names', 'discoverable']
+    form_class = DiscoveryForm
+
+    def form_valid(self, form):
+        coor_values = []
+        coordinate_dictionary = []
+        self.queryset = form.search()
+        if len(self.request.GET):
```

Comment:
This is for generating json data which is used in map view. It does nothing for the list view data.